src/Tribe/Ticket_Object.php

```diff
@@ -272,6 +298,43 @@ public function stock( $value = null ) {
 		}
 
 		/**
+		 * Sets of gets the current global stock mode in effect for the ticket.
+		 *
+		 * Typically this is one of the constants provided by Tribe__Tickets__Global_Stock:
+		 *
+		 *     GLOBAL_STOCK_MODE if it should draw on the global stock
+		 *     CAPPED_STOCK_MODE as above but with a limit on the total number of allowed sales
+		 *     OWN_STOCK_MODE if it should behave as if global stock is not in effect
+		 *
+		 * @param string $mode
+		 *
+		 * @return string
+		 */
+		public function global_stock_mode( $mode = null ) {
+			if ( is_string( $mode ) ) {
+				$this->global_stock_mode = $mode;
+			} elseif ( null === $mode ) {
+				return $this->global_stock_mode;
```

Comment:
Is there any reason to avoid always returning this regardless of what the value of `$mode` is?